app/LinkLiteManager/OptionsModels/RquestPollingServiceOptions.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace LinkLiteManager.OptionsModels
+{
+    public class RquestPollingServiceOptions
+    {
+        /// <summary>
+        /// Polling interval in seconds
+        /// for fetching queries from the RQUEST Connector API
+        /// </summary>
+        public int QueryPollingInterval { get; set; } = 5;
+
+        /// <summary>
+        /// 
+        /// </summary>
+        public string RquestCollectionId { get; set; } = "";
```

Comment:
🔧 this should now be irrelevant, as we don't configure the collection id through appsettings etc now - we use ActivitySources instead?

Reply:
I set it to use the first row of an ActivitySource for now, since we only have one resource id currently. How do we want to specify which one should be polled in the future?

Reply:
the polling behaviour for multiple sources is a bit TBD at the moment, but I suspect to begin with we will poll for all of them, and see how it goes.

What you've done currently works for me.